llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -416,7 +416,10 @@ struct IndexedMemProfRecord {
   // the last entry in the list with the same function GUID.
   llvm::SmallVector<CallStackId> CallSiteIds;
 
-  void clear() { AllocSites.clear(); }
+  void clear() {
```

Comment:
An more durable alternative to this:
`*this = IndexedMemProfRecord();`

Up to you if you want to adopt it.

Reply:
I like your suggestion much better.  Thanks!